lib/schema.js

```diff
@@ -37,12 +44,18 @@ module.exports = [
     filters: {
       projection: {
         title: 1,
-        slug: 1,
-        type: 1,
-        tags: 1
+        _url: 1
       }
     }
   },
+  {
+    name: '_linkFile',
+    label: 'Link File',
+    type: 'joinByOne',
+    withType: 'apostrophe-file',
+    idField: 'fileId',
```

Comment:
I would leave out `idField` and let it be set automatically. Since it is optional to specify it, most developers - if they had an obscure reason to care about it - would guess that it is `linkFileId` because that is what you'll get automatically.

Reply:
(If however this branch is already in use in some project prior to npm release then I could certainly live with it to avoid a bc problem)

Reply:
I withdraw this request because this module is in use in production (via git dependencies) and so changing the idField, which isn't harmful, would be a bc break even if slightly more intuitive in some rare situation.